File: Day6/Program1.c
. Write a C Program to Generate First N Triangular Numbers.

#include <stdio.h> 
void triangular_series(int n) 
{ 
    for (int i = 1; i <= n; i++)  
        printf(" %d ", i*(i+1)/2); 
} 
  	
// Driven Function 
int main() 
{ 
    int n;
    printf("enter the number:");
    scanf("%d",&n);
    triangular_series(n); 
    return 0; 
}

File: 25-06-2020/Program1.py
1. Write a python program for cube sum of first n natural numbers.


def sumOfSeries(n): 
	sum = 0
	for i in range(1, n+1): 
		sum +=i*i*i 
	return sum
n = int(input("Enter number:"))
print(sumOfSeries(n)) 
